packages/dds/tree/api-report/tree.api.md

```diff
@@ -1232,6 +1228,9 @@ export class LeafNodeStoredSchema extends TreeNodeStoredSchema {
 export interface LocalNodeKey extends Opaque<Brand<SessionSpaceCompressedId, "Local Node Key">> {
 }
 
+// @public
+export const makeFieldSchema: <Kind extends FieldKind = FieldKind, Types extends ImplicitAllowedTypes = ImplicitAllowedTypes>(kind: Kind, allowedTypes: Types, props?: FieldProps) => FieldSchema<Kind, Types>;
```

Comment:
the motivation for of this change was to allow us to keep this flexible construction API which can make any possible FieldSchema (even ones we don't want people making) internal to the package, while users of the package public API have to use one of the less flexible options from SchemaFactory. Thus we want to avoid marking this public: instead it shouldn't even be package exported: we should only need access to it to implement SchemaFactory.

Reply:
Removed package exports and public tag